dashboard/test/ui/features/spritelab/spritelab.feature

```diff
@@ -1,3 +1,4 @@
+@no_circle
```

Comment:
This first tag is sufficient - skipping the feature implicitly skips the scenarios beneath it.